todo/models.py

```diff
@@ -2,5 +2,27 @@
 from __future__ import unicode_literals
 
 from django.db import models
+from django.contrib.auth.models import User
 
 # Create your models here.
+
+
+class Task(models.Model):
+    STATUS_CHOICES = [("todo", "todo"), ("in_progress", "in_progress"), ("done", "done"), ("to_delete", "to_delete")]
```

Comment:
http://anthonyfox.io/2017/02/choices-for-choices-in-django-charfields/, option #2 for now